ui-tests/cypress/support/restAPI/contentsAPI.js

```diff
@@ -33,7 +33,7 @@ class ContentsController {
       body: [{
         ...content
       }]
-    });
+    }).then(response => ({ controller: this, response }));
```

Comment:
you can append the _then_ where you need it, in order to leave to each use case what to do with the result of the request

Reply:
make sense. I have made changes